instruments/tektronix/tektds5xx.py

```diff
@@ -500,7 +499,7 @@ def trigger_coupling(self):
 
         :type: `TekTDS5xx.Coupling`
         """
-        return TekTDS5xx.Coupling[self.query("TRIG:MAI:EDGE:COUP?")]
+        return TekTDS5xx.Coupling(self.query("TRIG:MAI:EDGE:COUP?"))
```

Comment:
again, name versus value error. The instrument surely gives the value back, not the name of the enum.